label_studio/data_manager/migrations/0008_manual_counters_update.py

```diff
@@ -34,7 +34,13 @@ def backwards(apps, schema_editor):
 class Migration(migrations.Migration):
     atomic = False
 
-    dependencies = [('data_manager', '0007_auto_20220708_0832'), ('tasks', '0023_auto_20220620_1007'), ('core', '0001_initial'), ('projects', '0017_project_pinned_at')]
+    dependencies = [
+        ('data_manager', '0007_auto_20220708_0832'),
+        ('tasks', '0023_auto_20220620_1007'),
+        ('core', '0001_initial'),
+        ('projects', '0017_project_pinned_at'),
+        ('organizations', '0004_organization_contact_info'),
```

Comment:
@juliosgarbi the issue was that this migration is somewhat special - it's handwritten, and it reads `Organization` model instances using ORM methods from the model class. Because the `Organization` model code now expects `contact_info` to be defined on each row of the underlying table, without this new dependency on the migration that adds `contact_info`, this migration would fail upon execution.

A weird and unexpected problem :joy: but should be good now.